### DISCUSSION TITLE
Issue for multiple tabs in a page

### DISCUSSION THREAD

**Discussion by cridesi06** [orchidsoftware/platform#2571] [General]
I have a page that contain more Layout::tabs and in every tab there is a list layout, the problem is that all of those tab have a column called "name" and when i filter by this column, if i filter in the last tab the filter is duplicated in every tabs, if i filter in the others tab (not in the last one) the filter doesn't work.
I just asked for this issue and they answered me to build a Menu, but i don't need a menu, i need more tabs in a single page.

**Comment by plamenradev** [comment]
You can use TabMenu, more information here:
https://orchid.software/en/docs/custom-template/#tabmenu
You have the same look and feel like using the regular tabs.
